cactus_test_definitions/client/procedures/ALL-28.yaml

```diff
@@ -245,7 +241,6 @@ Steps:
           duration_seconds: 300
           opModExpLimW: $(setMaxW * 0.3)
           opModImpLimW: $(setMaxW * 0.3)
-          ramp_time_seconds: 60
```

Comment:
The test procedure asserts that we ramp up to last 3kW export limit at the 1% ramp rate from the DefaultDERControl:

<img width="750" height="220" alt="Screenshot 2025-12-04 at 13 46 01" src="https://github.com/user-attachments/assets/7edf24ee-d870-4f0b-bdee-f0e6bcf2ffc1" />